Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -208,10 +216,12 @@ public struct PackageBuilder {
     }
 
     /// Returns path to all the items in a directory.
+    /// FIXME: This is generic functionality, and should move to FileSystem.
```

Comment:
I feel that I need to go and look into where rerooted file system ended up before commenting more.  I may have the wrong impression about how it's turned out, I've always considered it a layer in a file system stack.